examples/plot_03_competing_risks.py

```diff
@@ -0,0 +1,250 @@
+"""
+==============================
+Exploring the accuracy in time
+==============================
+
+In this notebook, we showcase how the accuracy in time metric is defined, behaves, and
+how to interpret it.
+"""
+
+# %%
+# Definition of the Accuracy in Time
+# ==================================
+# Here is a little bit of context about this metric introduced in [Alberge2024]_:
+#
+# - It is a generalization of the accuracy metric in the survival and the competing
+#   risks setting, which is the proportion of correct predictions (e.g. highest
+#   predicted probability) at a fixed time.
+# - It is computed for different time horizons, given by the user, with direct time
+#   stamps or quantiles of the observed durations.
+# - For a given patient at a fixed time, we compare the actual observed event to the
+#   most predicted one. The censored patients are removed from the computation after
+#   their censoring duration. In other word, let's imagine a patient who has
```

Comment:
there is a logic problem in the sequence, the sentence is about censoring, and in other worlds suggests we will explicit something about censoring.

I suggest
"For a given patient at a fixed time, we compare the actual observed event to the most predicted one. In other words ...... highest probability. The censored patients are removed from the computation after their censoring duration."